octoprint_octorelay/__init__.py

```diff
@@ -28,6 +28,11 @@ class OctoRelayPlugin(
     octoprint.plugin.RestartNeedingPlugin
 ):
 
+    def __init__(self):
+        self.polling_timer = None
+        self.model = None
```

Comment:
I'm not sure about safety of this statement because of this issue in the past:
https://github.com/borisbu/OctoRelay/issues/25

The presence of `model` property (which is set by `on_after_startup`) is a condition involved in `on_event`:

```py
    def on_event(self, event, payload):
        self._logger.debug("Got event: {}".format(event))
        if event == Events.CLIENT_OPENED:
            if hasattr(self, 'model'):
                self.update_ui()
```

Reply:
It seems that in some cases, `on_event` can be triggered before `on_after_startup`

Reply:
fixed in 68ecafd6cd0a697aa9bf99ae6ac9966ea9214210 by constructing as dictionaries